packages/nux/src/components/dot-tip/style.scss

```diff
@@ -12,7 +12,8 @@ $dot-scale: 3;  // How much the pulse animation should scale up by in size
 
 	&::before {
 		animation: nux-pulse 1.6s infinite cubic-bezier(0.17, 0.67, 0.92, 0.62);
-		background: rgba($blue-medium-800, 0.9);
+		background: rgba(#00739c, 0.9);
```

Comment:
I wonder if this should use the theme colors (css vars)

Reply:
It should. But I don't know where we use these anymore, so I wasn't able to test this one, and it looks as if the rgba color opacity is animated, which we can't easily do with the CSS variables — right? So I left it alone for now.

Reply:
it's a deprecated component, so not very important for sure.